CHANGELOG.md

```diff
@@ -1,20 +1,52 @@
 # Azure Kubernetes Service Changelog
 
+
+
+## Release 2023-08-20
+
+Monitor the release status by regions at [AKS-Release-Tracker](https://releases.aks.azure.com/).
+
+### Announcements
+
+* No new clusters can be created with [Azure AD Integration (legacy)](https://learn.microsoft.com/azure/aks/azure-ad-integration-cli). Existing AKS clusters with Azure Active Directory integration will keep working. All Azure AD Integration (legacy) AKS clusters will be migrated to [AKS-managed Azure AD](https://learn.microsoft.com/azure/aks/managed-azure-ad) automatically starting from 1st Dec. 2023. We recommend updating your cluster with AKS-managed Azure AD before 1 Dec 2023. This way you can manage the API server downtime during non-business hours.
+* Please review the following CVEs and instructions in the linked issues to see if these vulnerabilities impact your cluster - [CVE-2023-3676](https://github.com/Azure/AKS/issues/3869), [CVE-2023-3955](https://github.com/Azure/AKS/issues/3870), and [CVE-2023-3893](https://github.com/Azure/AKS/issues/3871). If your clusters are impacted, please follow mitigation releated updates on these issues.
```

Comment:
mispelled related at the end of the sentence.

I would also just change the sentence to.  All windows nodes are affected with the following CVE's, please update your windows node to the VHD version 230809 which is detailed out in the following github articles.